src/main/c/org_broadinstitute_hellbender_utils_fermi_FermiLiteAssembler.c

```diff
@@ -7,82 +7,102 @@
 JNIEXPORT jobject JNICALL
 Java_org_broadinstitute_hellbender_utils_fermi_FermiLiteAssembler_createDefaultOptions( JNIEnv* env, jclass cls ) {
     fml_opt_t* pOpts = malloc(sizeof(fml_opt_t));
+    if ( !pOpts ) return 0;
     fml_opt_init(pOpts);
-	return (*env)->NewDirectByteBuffer(env,pOpts,sizeof(fml_opt_t));
+    return (*env)->NewDirectByteBuffer(env,pOpts,sizeof(fml_opt_t));
+}
+
+static void freeReads( bseq1_t* pReads, bseq1_t* pEnd ) {
+    bseq1_t* pRead;
+    for ( pRead = pReads; pRead != pEnd; ++pRead ) {
+        free(pRead->seq);
+        free(pRead->qual);
```

Comment:
It seems like this function could get called before `seq` and `qual` are allocated for every `bseq1_t`, in which case you could call `free()` on memory that was never `malloc'd`. If the switch from `malloc()` to `calloc()` below was intended to address this issue, since `free(NULL)` is safe, then add a comment to that effect above the `calloc()` call on line 32.